debian/mariadb-client-10.5.install

```diff
@@ -1,29 +1,17 @@
 debian/additions/innotop/innotop usr/bin/
 debian/additions/mysqlreport usr/bin/
 usr/bin/mariadb-conv
-usr/bin/mysql_find_rows
-usr/bin/mysql_fix_extensions
-usr/bin/mysql_waitpid
-usr/bin/mysqlaccess
-usr/bin/mysqladmin
-usr/bin/mysqldump
-usr/bin/mysqldumpslow
-usr/bin/mysqlimport
-usr/bin/mysqlshow
+usr/bin/mariadb-find-rows
+usr/bin/mariadb-fix-extensions
+usr/bin/mariadb-waitpid
+usr/bin/mariadb-access
+usr/bin/mariadb-admin
```

Comment:
list is not alphabetical, just run `wrap-and-sort -a -v` and it will fix itself automatically

Reply:
I ran wrap-and-sort -a -v in new commit cf70893